sdk/spring/azure-spring-boot-test-keyvault/src/test/java/com/azure/test/keyvault/KeyVaultIT.java

```diff
@@ -131,30 +127,23 @@ public void keyVaultWithAppServiceMSI() {
     }
 
     @Test
-    @Ignore
-    public void keyVaultWithVirtualMachineMSI() throws Exception {
+    public void keyVaultWithVirtualMachineMSI() {
         final VirtualMachine vm = AZURE.virtualMachines().getByResourceGroup(SPRING_RESOURCE_GROUP, VM_NAME);
 
         final String host = vm.getPrimaryPublicIPAddress().ipAddress();
 
-        // Upload app.jar to virtual machine
-        final MavenBasedProject app = new MavenBasedProject("../azure-spring-boot-test-application");
-        app.packageUp();
-
-        final File file = new File(app.artifact());
-
-        if (!file.exists()) {
-            throw new FileNotFoundException("There's no app.jar file found.");
-        }
-        try (SSHShell sshShell = SSHShell.open(host, 22, VM_USER_USERNAME, VM_USER_PASSWORD);
-            FileInputStream fis = new FileInputStream(file)) {
-            LOGGER.info("Uploading jar file...");
-            sshShell.upload(fis, "app.jar", "", true, "4095");
-        }
-
-        // run java application
         final List<String> commands = new ArrayList<>();
         commands.add(String.format("cd /home/%s", VM_USER_USERNAME));
+        commands.add("mkdir azure-sdk-for-java");
+        commands.add("cd azure-sdk-for-java");
+        commands.add("git init");
+        commands.add("git remote add origin https://github.com/Azure/azure-sdk-for-java.git");
+        commands.add("git config core.sparsecheckout true");
+        commands.add("echo sdk/spring > .git/info/sparse-checkout");
+        commands.add("git pull origin master");
```

Comment:
Can we use current branch instead of `master` branch?

Refs: https://stackoverflow.com/questions/49106104/get-current-git-branch-inside-a-java-test